cli/cmd/deploy.go

```diff
@@ -22,28 +22,28 @@ import (
 var deployCmd = &cobra.Command{
 	Use:   "deploy",
 	Short: "Deploy a new connector",
-	Long: `Deploy a new connector or replace the old version if it alrerady exists.
+	Long: `Deploy a new connector or replace the old version if it already exists.
 	This command is executes all its steps synchronously.
 	flags:
 		--url -u: url of the kafka-connect server
-		--file -f: path to the config file
+		--file -f: path to the config file or folder containing config files
```

Comment:
As a user, I would expect better clarity both in the description and the parameters naming. I won't search a parameter `file` if I want to put a folder. Maybe rework this part.
What about `-i --input` or even separate `--file and --folder`?